ROADMAP.md

```diff
@@ -1,41 +1,42 @@
-## Teams Toolkit Roadmap
+# Teams Toolkit Roadmap
 
 Our roadmap and priorities are shaped by your feedback, issues, discussions, surveys, social media, and market opportunities. We want to make building apps for Teams fast, simple, and delightful. We're constantly learning, and we listen to your feedback and adapt our plans if needed. The items below may not include everything we have planned. If you have feedback or suggestions, create a new GitHub issue, or react with a 👍 on an existing issue.
 
-Based on your feedback, we plan to focus on the following themes in 2022:
+Teams Toolkit continuously focuses on:
 
-**Microsoft 365 Platform:** We’re committed to providing Day 1 support for new Microsoft 365 capabilities and features.
+- **Microsoft 365 Platform:** We’re committed to providing Day 1 support for new Microsoft 365 capabilities and features.
 
-**Support for familiar language & tools:** Teams Toolkit works with the languages, frameworks, tools, and services you know and love.
+- **Support for familiar language & tools:** Teams Toolkit works with the languages, frameworks, tools, and services you know and love.
 
-**Get started fast:** Teams Toolkit is the simplest way to get started building for Microsoft 365 with delightful tooling, samples, and documentation.
+- **Get started fast:** Teams Toolkit is the simplest way to get started building for Microsoft 365 with delightful tooling, samples, and documentation.
 
-**Development velocity:** best-in-class productivity to create, build, and deliver your apps.
+- **Development velocity:** best-in-class productivity to create, build, and deliver your apps.
 
-**Ship with confidence:** Teams Toolkit is enterprise-ready to help developers and IT administrators deliver, manage, and monitor apps with confidence.
+- **Ship with confidence:** Teams Toolkit is enterprise-ready to help developers and IT administrators deliver, manage, and monitor apps with confidence.
 
-#### Q3-Q4 2022
+## Q3-Q4, 2023
 
-##### Teams Toolkit General Availability
-We’re planning to make Teams Toolkit for Visual Studio generally available later this year in Visual Studio 2022.
+- **Extend Microsoft 365 Copilot**: Teams Toolkit for Visual Studio and Visual Studio Code will soon let users quickly build a plugin to extend Microsoft 365 Copilot. This [demonstration](https://www.youtube.com/watch?v=6ZNi1GDxvf0&ab_channel=Microsoft365Developer) shows how simple it will be to build a plugin to extend Microsoft 365 Copilot using the Teams Toolkit extension in Visual Studio Code. This is a preview of the experience you will have when we release our first public preview coming soon.
+- **Continuous improvements of templates and samples**: App templates and samples continue to be the key starting point for developers to learn how to build apps for Teams and Microsoft 365. We will continue to simplify the app templates and improve the usability of sample app gallery as we incrementally adding more samples to it.
```

Comment:
Is "Teams and Microsoft 365" supposed to be "Microsoft 365 including Teams"?

Reply:
Updated.